src/rebar_git_resource.erl

```diff
@@ -124,7 +124,7 @@ parse_git_url(not_scp, Url) ->
     case rebar_uri:parse(Url, UriOpts) of
         #{path := Path, host := Host} ->
             {ok, {Host, filename:rootname(Path, ".git")}};
-        {error, Reason} ->
+        {error, _, Reason} ->
```

Comment:
This is a subtle difference, but it's an attempt at making the return types compatible between `uri_string:parse` and `http_uri:parse`, as expressed in `rebar_uri:parse/2`.